server/storage/endpoint/key_path.go

```diff
@@ -136,3 +141,30 @@ func KeySpaceSafePointPrefix() string {
 func KeySpaceGCSafePointSuffix() string {
 	return "/" + keySpaceGCSafePointSuffix
 }
+
+// KeyspaceMetaPrefix returns the prefix of keyspaces' metadata.
+// Prefix: keyspaces/meta/
```

Comment:
It seems we use `key_space` as prefix when we store safepoint `/key_space/gc_safepoint/`
so could and should we  share the same prefix?

Reply:
It will be changed in a later PR that moves keyspace safepoint management under keyspace manager